### PR TITLE
Add CORS config / backend Headers for WebWorker/SharedArrayBuffer support

### PR DESCRIPTION
**Pull request by achimcc**
- Adds a middleware to the Rust Backend Webserver 

- Serves the correct headers for QWebWorker support in modern Browsers

### CODE REVIEW

File: crates/backend/src/main.rs [hunk 1]
```diff
@@ -24,11 +25,17 @@ use actix_web::{
 async fn main() -> std::io::Result<()> {
     let serve_from = "../../packages/playground/dist";
     if !Path::new(serve_from).is_dir() {
-        panic!("{} is not a valid directory", serve_from);
+        panic!("{} is not a valid directory \n Hint: try to run yarn build inside the Frontend Repo first.", serve_from);
```
Comment (thought2) [inline crates/backend/src/main.rs:28]: I really think we should keep services as independent from the environment as possible. This error message introduces knowledge of a specific environment. I think we can stay with the existing one and handle the DX at another level.

File: crates/backend/src/main.rs [hunk 2]
```diff
@@ -24,11 +25,17 @@ use actix_web::{
 async fn main() -> std::io::Result<()> {
     let serve_from = "../../packages/playground/dist";
```
Comment (thought2) [inline crates/backend/src/main.rs:26]: Same as below, but it will be handled by https://github.com/paritytech/ink-playground/pull/31

**Review by thought2 — CHANGES_REQUESTED** [review]

File: crates/backend/src/main.rs (hunk 1 above)
Comment (achimcc) [reply crates/backend/src/main.rs]: Done

**Review by thought2 — APPROVED** [review]
thx!

**Pull request merged**